cfme/intelligence/chargeback/assignments.py

```diff
@@ -68,23 +69,17 @@ class Assign(Updateable, Pretty, Navigatable):
 
     Usage:
         enterprise = Assign(
```

Comment:
Required: this is not accurate, as we use collections now, so the usage should be like appliance.collecions.assignments.instantiate(assign_to, tag_category)